src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsoffendercategorisationapi/config/OffenderCategorisationApiConfiguration.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsoffendercategorisationapi.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import java.time.Clock
+
+@Configuration
+class OffenderCategorisationApiConfiguration {
+  @Bean
+  @Profile("!test")
+  fun clock(): Clock? = Clock.systemDefaultZone()
```

Comment:
Returns:
a clock that uses the best available system clock in the default zone, not null
```suggestion
  fun clock(): Clock = Clock.systemDefaultZone()
```